04.md

```diff
@@ -40,13 +44,50 @@ let event = {
   created_at: Math.floor(Date.now() / 1000),
   kind: 4,
   tags: [['p', theirPublicKey]],
-  content: encryptedMessage + '?iv=' + ivBase64
+  content: encryptedMessage + '??' + ivBase64 + "??1"
```

Comment:
it's actually faster than having ?iv=, ?ver=.  benchmark it.

Reply:
I'm talking about algorithmic complexity in lower-level languages. A parser will need to walk through every ciphertext character and stop on `??`.

Why? We already have JSON.

Reply:
json is no faster.   and "?" never appears in base64.   this is always going to be faster than JSON.   i will change it to one "?".   it will always be order N no matter what.

Reply:
updated to a single delimiter.